src/pages/HomePage/HomePageComponents/Banner/Banner.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import '../../../../Colours/colours.css';
```

Comment:
You can make use of absolute paths.
Read more about it from here. https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160